monkey/tests/data_for_tests/monkey_configs/default_config.py

```diff
@@ -0,0 +1,119 @@
+            "options": {
+                "encryption": {
+                    "enabled": True,
+                    "directories": {"linux_target_dir": "", "windows_target_dir": ""},
+                },
+                "other_behaviors": {"readme": True},
+            },
+        }
+    ],
+    "custom_pbas": {
+        "linux_command": "",
+        "linux_filename": "",
+        "windows_command": "",
+        "windows_filename": "",
+    },
+    "propagation": {
+        "network_scan": {
+            "tcp": {
+                "timeout": 3000,
+                "ports": [
+                    22,
+                    80,
+                    135,
+                    443,
+                    445,
+                    2222,
+                    3306,
+                    3389,
+                    5985,
+                    5986,
+                    7001,
+                    8008,
+                    8080,
+                    8088,
+                    8983,
+                    9200,
+                    9600,
+                ],
+            },
+            "icmp": {"timeout": 1000},
+            "fingerprinters": [
+                {"name": "elastic", "options": {}},
+                {
+                    "name": "http",
+                    "options": {"http_ports": [80, 443, 7001, 8008, 8080, 8983, 9200, 9600]},
+                },
+                {"name": "mssql", "options": {}},
+                {"name": "smb", "options": {}},
+                {"name": "ssh", "options": {}},
+            ],
+            "targets": {
+                "blocked_ips": [],
+                "inaccessible_subnets": [],
+                "local_network_scan": True,
+                "subnets": [],
+            },
+        },
+        "maximum_depth": 2,
+        "exploitation": {
+            "options": {"http_ports": [80, 443, 7001, 8008, 8080, 8983, 9200, 9600]},
+            "brute_force": [
+                {
+                    "name": "MSSQLExploiter",
+                    "options": {},
+                },
+                {
+                    "name": "PowerShellExploiter",
+                    "options": {},
+                },
+                {
+                    "name": "SSHExploiter",
+                    "options": {},
+                },
+                {
+                    "name": "SmbExploiter",
+                    "options": {"smb_download_timeout": 30},
+                },
+                {
+                    "name": "WmiExploiter",
+                    "options": {"smb_download_timeout": 30},
+                },
+            ],
+            "vulnerability": [
+                {
+                    "name": "HadoopExploiter",
+                    "options": {},
+                },
+                {
+                    "name": "Log4ShellExploiter",
+                    "options": {},
+                },
+            ],
+        },
+    },
+}
+
+DEFAULT_CONFIG = AgentConfiguration.from_mapping(flat_config)
```

Comment:
Can we create this from schema as well? Why is there a need for different default config in tests compared to `monkey/common/configuration/default_agent_configuration.py`

Reply:
Nothing uses this, so I removed it.